docs/docs/reference/release-notes/v2-maintenance-plan.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gatsby Framework Version Support
+---
+
+## Overview
+
+As we continue evolving the Gatsby framework, and as we move from one major version to the next, we will communicate the duration and types of support provided to the prior major version. This allows users to effectively plan their Gatsby framework upgrades.
+
+## What is happening, when?
+
+Gatsby Version 2.32 will enter Maintenance Long Term Support on **June 8, 2021**. At that same time, Gatsby Version 3 will be considered the Active Long Term Support version of Gatsby. The Maintenance Long Term Support will continue until the release of Gatsby v4 (date TBD).
+
+## What does this mean to users?
+
+- As of June 8, 2021, Gatsby version 2.32 will only receive critical patches.
```

Comment:
This feels like it needs another sentence to explain what the implications are for the user.

e.g. "Gatsby 2.32 will only receive critical patches, which means that we won't be accepting new issues for that version." (or whatever the boundaries are.) How will the user's normal behavior patterns be affected?

Reply:
It might also be worth it to say something like "if you have a problem with v3, you can still open an issue." (Although that might fit better in the "critical patches" section below.)

Reply:
Good points. Never hurts to reassure and remind that / where issues can be reported.

Reply:
I added callouts here to issues / support area of .com as well.

Also, since I explain in the `What is happening, when` section that v3 is required for feature enhancements, I didn't see the need to reiterate that v2 wouldn't be receiving new features.